src/cpp/statistics/fastdds/publisher/qos/DataWriterQos.cpp

```diff
@@ -32,7 +32,7 @@ DataWriterQos::DataWriterQos()
     durability().kind = eprosima::fastdds::dds::TRANSIENT_LOCAL_DURABILITY_QOS;
     publish_mode().kind = eprosima::fastdds::dds::ASYNCHRONOUS_PUBLISH_MODE;
     history().kind = eprosima::fastdds::dds::KEEP_LAST_HISTORY_QOS;
-    history().depth = 100;
+    history().depth = 1;
```

Comment:
This will need a documentation update. It can be done at the same time than the one I requested [here](https://github.com/eProsima/Fast-DDS/pull/2176#discussion_r707390209)